CMakeLists.txt

```diff
@@ -324,6 +333,11 @@ if(OCOS_ENABLE_MATH)
   endif()
 
   file(GLOB TARGET_SRC_MATH "operators/math/*.cc" "operators/math/*.h*")
+  if(OCOS_USED_CUDA)
+    file(GLOB TARGET_SRC_MATH_CUDA "operators/math/cuda/*.*")
```

Comment:
do we need to add onnxruntime-gpu to requirements.txt or does ort take care of that if we use CUDA kernels?

Reply:
that's Python support issue.
Moving forward, we don't specify any dependency on ort or ort-gpu since we don't know which one the user's preference is.